grammars/css.cson

```diff
@@ -90,14 +99,14 @@
     ]
   }
   {
-    'begin': '(?=^\\s*@media(\\s+|$))'
+    'begin': '(?=@media(\\s+|$))'
```

Comment:
Yes but I'm not very sure how to fix it. Should I loose the requirement for `@media` to be followed by at least a space? I could do `(?=@media\\b))`. What do you think?
There are so many issues with comments in this grammar...